index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="dist/style/main.css" />
+    <title>Typing</title>
+  </head>
+  <body>
+    <section class="landing-page">
+      <div class="logo">Typing</div>
+
+      <a href="#sign_in-box" id="sign_in-btn" class="sign_in-btn">Sign In</a>
+
+      <div class="content">
+        <h1 class="heading-primary">
+          Welcome To The Best Typing Websie In The World
+        </h1>
+      </div>
+    </section>
+
+    <section class="sign_in">
+      <form class="sign_in--form">
+        <div class="group-input">
```

Comment:
I think `input-group` is a better name, what do you think?
```suggestion
        <div class="input-group">
```